packages/contracts/contracts/QuadraticVoteCollectModule.sol

```diff
@@ -8,44 +8,28 @@ import { Errors } from "./libraries/Errors.sol";
 import { FeeModuleBase } from "./FeeModuleBase.sol";
 import { ModuleBase } from "./ModuleBase.sol";
 
-import { FollowValidationModuleBase } from "./FollowValidationModuleBase.sol";
-
 import "./interfaces/IRoundImplementation.sol";
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import { IERC721 } from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
-import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
-import "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
-
-
 /**
  * @notice A struct containing the necessary data to execute collect actions on a publication.
  *
- * @param amount The collecting cost associated with this publication.
  * @param currency The currency associated with this publication.
- * @param collectLimit The maximum number of collects for this publication. 0 for no limit.
- * @param currentCollects The current number of collects for this publication.
- * @param recipient The recipient address associated with this publication.
  * @param referralFee The referral fee associated with this publication.
  * @param followerOnly True if only followers of publisher may collect the post.
  * @param endTimestamp The end timestamp after which collecting is impossible. 0 for no expiry.
  */
 struct ProfilePublicationData {
```

Comment:
I tried to keep all the original data being passed into the profilePublication and just add the addresses we needed.  Is there a reason we don't need to do this?